README.md

```diff
@@ -1045,6 +1045,9 @@ Table of Contents
   * [js.org](https://js.org) - Free js.org subdomains for GitHub Pages for the JavaScript community.
   * [pp.ua](https://nic.ua/) — Free pp.ua subdomains.
   * [thedev.id](https://thedev.id) — A free thedev.id subdomain for developers.
+  * [js.cool](https://js.cool) - Free js.cool subdomain for developers.
```

Comment:
This has a requirement `Your site must have a Chinese translation edition` which I would say disqualifies it.

Reply:
alright